packages/web/sentry.client.config.ts

```diff
@@ -0,0 +1,30 @@
+// This file configures the initialization of Sentry on the client.
+// The config you add here will be used whenever a users loads a page in their browser.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "https://c696452bb7ce4cc98150142ebea1c32f@o4505285755600896.ingest.sentry.io/4505285757698048",
```

Comment:
Is it ok for this to be a hardcoded URL? I'm not opposed, in fact I prefer it since it's one less dependency. But if it's expected to change often, or across envs, IMO it should be an env var.

Reply:
> Let's make sure not to merge until we pay our sentry bill

It's a paid account now 👍  cc: @mattupham 

I'll monitor the usage and upgrade as needed.